.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: master
+on:
+  push:
+    branches:
+      - master
+permissions:
+  contents: read
+  id-token: write
+concurrency:
+  cancel-in-progress: false
+  group: "${{ github.workflow }}-${{ github.ref }}"
+jobs:
+  upload-aws-backup-elastio-integration:
+    runs-on:
+      - ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: "actions/checkout@v3"
+        with:
+          fetch-depth: '30'
+          token: "${{ secrets.SECRET_TOKEN }}"
+      - name: Check if files were modified
+        id: changes
+        uses: "dorny/paths-filter@v2"
+        with:
+          filters: |
+            filechange:
+              - 'aws-backup-elastio-integration/**'
+      - name: Configure AWS credentials
+        if: "steps.changes.outputs.filechange == 'true'"
+        uses: "aws-actions/configure-aws-credentials@v3"
+        with:
+          aws-region: "${{ secrets.CI_AWS_DEFAULT_REGION }}"
+          role-to-assume: "${{ secrets.CI_AWS_ROLE_ARN}}"
+          special-characters-workaround: 'true'
+      - name: Upload lambda and cfn to bucket
+        if: "steps.changes.outputs.filechange == 'true'"
+        env:
+          NAME: aws-backup-elastio-integration
+          CFN_NAME: Elastio_stack.yaml
+          S3_BUCKET: "${{ secrets.CI_S3_BUCKET }}"
```

Comment:
This must not be a secret. Let's use `vars` instead. Or does it even have to be specified outside of the repo? Could we just hardcode the bucket name?

Reply:
As this is public repo I thought about putting it away. But yeah, since we have a url in cfn plaintext, probably we could do it.

Reply:
Updated it.